public/src/components/channelManagement/campaigns/CampaignsEditor.tsx

```diff
@@ -136,6 +179,19 @@ function CampaignsEditor({ campaign }: CampaignsEditorProps): React.ReactElement
     }
   };
 
+  const updateDescription = ({ description }: FormData): void => {
```

Comment:
having separate handlers for each field is actually causing it to overwrite earlier changes, because it's fighting against react-hook-form here. Do we even need to use react-hook-form in this component?

Reply:
It took some doing (MUI Switch is not a nice component!) ... but fixed. All three fields now use the the same `onSubmit` function.

Reply:
For future reference - we're pretty tied to using RHF if we're using our RTE component. Also of interest, we're using Remirror v1.0.x but the repo has progressed a lot over the past year and is now on v2.0.x ... it might be worth our while to investigate whether we should upgrade sooner rather than later?

Reply:
> we're pretty tied to using RHF if we're using our RTE component.
Why? It doesn't use RHF

Reply:
I couldn't get it to work without RHF collecting the data in the parent component